src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Media/Given_ImageBrushStretch.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions.Formatting;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Private.Infrastructure;
+using Uno.UI.RuntimeTests.Helpers;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Imaging;
+using static Private.Infrastructure.TestServices;
+using ImageBrush = Windows.UI.Xaml.Media.ImageBrush;
+
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Media
+{
+	[TestClass]
+	[RunsOnUIThread]
+	public class Given_ImageBrushStretch
+	{
+#if __SKIA__  || __MACOS__  // WASM does not have an equivalent of renderAsync  and Android behaves differently. See https://github.com/unoplatform/uno/pull/7238#issuecomment-937667565
+		[DataRow(Stretch.None)]
+		[DataRow(Stretch.Fill)]
+		[DataRow(Stretch.UniformToFill)]
+#if !__SKIA__ //Stretch.Uniform create a Mosaic in Skia which is not the desirable comportement 
+		[DataRow(Stretch.Uniform)]
+#endif
+		[DataRow(Stretch.None)]
+
```

Comment:
It looks like Stretch.None is duplicated and the test will run twice unnecessarily.